tests/e2e/tests/pilot/testdata/rbac/v1alpha1/istio-rbac-rules.yaml.tmpl

```diff
@@ -1,11 +1,12 @@
-# istio-rbac-rules.yaml to enforce access control to services
+# istio-rbac-rules.yaml to enforce access control for http services
```

Comment:
for http and TCP services? I saw you have TCP test cases in this file.

Reply:
Done. I forgot to mention the tcp here.